connectors/.env.example

```diff
@@ -0,0 +1,14 @@
+# Since the ".env" file is gitignored, you can use the ".env.example" file to
```

Comment:
to clean up - from https://create.t3.gg/ boilerplate.

Reply:
Yes let’s remove as much boilerplate as possible 👍